submissions/franqly/franqly.md

```diff
@@ -0,0 +1,99 @@
+# Nauvian Perspective: Franqly <author>stringweasel</author>
+
+The fastest speedrunner in the world - at time of writing this - is Franqly. He joined the Factorio speedrunner community recently with his first entry on [speedrun.com](https://www.speedrun.com/user/Franqly) only in October last year. However, today he is out-running legends like AntiElitz and Nefrums, and is the only player alive that has finished Factorio in [90 minutes flat](https://www.speedrun.com/factorio/run/y2xelq5m). In the time it takes you to watch the Bee Movie he can complete everything from crash landing on Nauvis to launching the rocket.
```

Comment:
> The fastest speedrunner in the world
>
> the only player alive that has finished Factorio in 90 minutes flat

Something to keep in mind here is that multiplayer runs exist - e.g. I have finished the game in less than an hour, together with 7 other runners. So, there should be some signifier that this is about singleplayer.

It's usually also better to limit statements like the very first sentence a bit, lest you get into arguing whether Franqly really is the fastest runner in general (what about categories besides any%, what about past game versions, what about stuff outside of speedrun.com etc). It takes away from the wow-factor of the intro sentence, but it also avoids a whole pile of unpleasant arguments.

Reply:
Yeah, that's a very good point. I've will see how I can adapt it.